package.json

```diff
@@ -20,40 +21,40 @@
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-loader": "^7.1.5",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-polyfill": "^6.23.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "clean-webpack-plugin": "^0.1.9",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "clean-webpack-plugin": "^0.1.19",
     "cookie": "^0.3.1",
     "cross-env": "^4.0.0",
     "dependency-graph": "^0.5.0",
-    "es6-promise": "^4.1.0",
+    "es6-promise": "^4.2.5",
```

Comment:
I don't know this project, just wondering what we support that requires the es6-promise polyfill

Reply:
Yep, that can go I think!